tests/hypervisor/test_esx.py

```diff
@@ -62,8 +65,7 @@ def test_type(self, virtwho, function_hypervisor, esx_assertion):
         # type option is disable but another config is ok
         new_file = '/etc/virt-who.d/new_config.conf'
         section_name = 'virtwho-config'
```

Comment:
Because several places call the `new_file` and `section_name`, so could we define them at the above of the Class as global values, for examples:
```
second_good_file = '/etc/virt-who.d/second_good_esx.conf'
second_good_section = 'second-virtwho-esx'
```

Reply:
Added them to `virtwho/__init__.py` file